Atomic/syscontainers.py

```diff
@@ -1954,6 +1954,15 @@ def _check_system_oci_image(self, repo, img, upgrade, src_creds=None):
             if not missing_layers:
                 return False
 
+        try:
+            d = util.Decompose(img)
+            if d.registry == "":
+                util.write_err("The image `{}` is not fully qualified.  The default registry configured for Skopeo will be used.".format(img))
```

Comment:
A nit, hopefully you won't mind :). It seems a bit weird to me that in this function, it is also possible to perform non-skopeo copy right? However, this code section seems to me that it only belongs to skopeo part? Do we possibly want to move it to the code block into the if statement later:

```
if can_use_skopeo_copy:

       try:
            xxxx
       except:
            pass
       
       return self._check_system_oci_image_skopeo_copy(repo, img, src_creds=src_creds)
```

Or may be we could reword the error message here to cover non-skopeo case?

Reply:
both code paths will end up in Skopeo, but in different ways.

The new way is to use "skopeo copy" that does basically everything: pulls the image, writes the references to the OSTree repository, check signatures.

The old way was to use "skopeo inspect", then check what layers we need and fetch them with "skopeo layers", that as you can see, it is even deprecated in newer Skopeo versions.

Reply:
ah, I see. Ignore my last comment then, did not notice the skopeo call in the codebase. Thanks for letting me know. :-)

Reply:
Do we need to respect the --quiet option?

Reply:
it seems we have `quiet` only for `atomic run` and `atomic images|containers list`, not other command is supporting it AFAICS:

```
# atomic pull --quiet foo
atomic: unrecognized arguments: --quiet

# atomic install --quiet foo
atomic: unrecognized arguments: --quiet
```

Or as a global option:

```
# atomic --quiet install foo
atomic: unrecognized arguments: --quiet
```

Reply:
We probably should add support for them.  ::)